docs/custom-features/external-environments.md

```diff
@@ -0,0 +1,25 @@
+# External Environment
+
+In the interest of security, the application is deployed to 2 separate environments: The "internal" and "external" environments.
+
+The `IS_EXTERNAL_ENV` environment variable (and setting) is used to switch the app to the "external" environment.
+
+## "Internal" environment
+
+The internal is private and only accessible to ONS staff.
+
+The Wagtail admin is only accessible on the "internal" environment.
+
+Management commands, database migrations, and other [scheduled jobs](./scheduled-jobs.md) are only executed in the internal environment.
+
+## External
+
+The "external" environment is the one used by the public.
+
+Writeable database access is restricted to a limited set of tables. This is enforced both at the database level and using database router (see `cms.core.db_router.ExternalEnvRouter`).
```

Comment:
```suggestion
Writeable database access is restricted to a limited set of tables. This is enforced both at the database level and by using a custom database router (see `cms.core.db_router.ExternalEnvRouter`).
```

Reply:
Fixed in https://github.com/ONSdigital/dis-wagtail/pull/236/commits/612559fc9ccbfd39730cf2d7193f202a2e968934